File: src/main/java/utils/HttpRequest.java
package utils;

import java.io.IOException;

import okhttp3.OkHttpClient;
import okhttp3.Request;
import okhttp3.Response;

public class HttpRequest {
    
    private static OkHttpClient client = new OkHttpClient();
    
    static public String getRequest(String url, String... values) throws IOException {
	
	StringBuilder s = new StringBuilder(url+"?");
	for(String val : values)
	    s.append(val+"&");
	
	Request request = new Request.Builder()
		.url(s.toString())
		.build();
	
	Response response =  client.newCall(request).execute();
	return response.body().string();
    }
    
    
}

File: src/main/kotlin/com/github/sylux6/watanabot/utils/HttpRequest.kt
package com.github.sylux6.watanabot.utils

import okhttp3.OkHttpClient
import okhttp3.Request
import java.io.IOException

@Throws(IOException::class)
fun getRequest(client: OkHttpClient, url: String, vararg values: String): String {

    val s = StringBuilder("$url?")
    for (`val` in values)
        s.append("$`val`&")

    val request = Request.Builder()
        .url(s.toString())
        .build()

    val response = client.newCall(request).execute()
    return response.body()!!.string()
}
